docker/Dockerfile

```diff
@@ -273,26 +262,10 @@ RUN cd $HOME/xformers && \
     CUDA_VISIBLE_DEVICES=0 pip3 install -e ".[dev]"
 
 # Install flash_attn
-RUN git clone https://github.com/jfc4050/flash-attention.git $HOME/flash-attention && \
-    cd $HOME/flash-attention/ && git checkout 3676bd2
-RUN cd $HOME/flash-attention/ && \
-    pip3 install -e ".[dev]"
-
-# Install epoi
-RUN git clone https://github.com/comaniac/epoi --recursive $HOME/epoi
-RUN cd $HOME/epoi && git fetch && git checkout fa90fa7 && pip3 install -e ".[dev]"
+RUN pip install flash-attn
```

Comment:
Have the changes Justin made upstreamed already?

Reply:
I followed @zarzen 's changes in the [ds-pipe](https://github.com/awslabs/slapo/commit/28b9fb3287d37f8c805b4d8777b5ef25b8c35e7d) branch. Have you tested whether the flash attention works?

Reply:
checked with others, they are using hash `https://github.com/jfc4050/flash-attention@7b46e19dc850b8ed70caf2f1b08728e2a191e6dc`